OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

```diff
@@ -148,6 +148,7 @@ public void TestGetOptimizelyConfigService()
                     "feat_with_var_test", new OptimizelyExperiment (
                         id: "11564051718",
                         key:"feat_with_var_test",
+                        "",
```

Comment:
i don't see OptimizelyConfigTest is in the project file, can you please check if this tests are running.